doc/sphinx/Makefile

```diff
@@ -41,6 +41,13 @@ copy-docs: clean-copy-docs
 	cp  ../release/quickReference.pdf     source/language/quickReference.pdf
 	cp  ../release/chapelLanguageSpec.pdf source/language/chapelLanguageSpec.pdf
 
+primers: clean-primers
+	@echo
+	@echo "Generating primers from "'$$CHPL_HOME'"/test/release/examples to "'$$CHPL_HOME'"/doc/sphinx/source/primers"
+	./chpl2rst.py ../../test/release/examples/primers/*.chpl --output=rst --prefix=source/primers
```

Comment:
The other relative paths are valid for a git repo and tarball release, these are the only ones that won't be